### DISCUSSION TITLE
Doubt

### DISCUSSION THREAD

**Discussion by HarishkumarSubramanian** [peaceiris/actions-gh-pages#899] [Q&A]
Whether the index.js is given in the repo? In action.yaml there is 
runs:
  using: 'node16'
  main: 'lib/index.js'
but in the repo there is no lib folder or index.js

**Comment by peaceiris** [comment]
The generated files are only on a tag.

e.g. https://github.com/peaceiris/actions-gh-pages/blob/v3.9.3/lib/index.js

cf.

- A script to release: https://github.com/peaceiris/actions-gh-pages/blob/main/release.sh
- A related comment: https://github.com/peaceiris/actions-gh-pages/issues/712#issuecomment-1059692405

**Reply by peaceiris** [reply]
For verifying the assets, we can use the following snippet.

```sh
git clone https://github.com/peaceiris/actions-gh-pages.git
cd ./actions-gh-pages
git checkout v3.9.3
nvm install
nvm use
npm i -g npm
npm ci
npm run build
git diff ./lib/index.js # we will get zero exit code
```
